examples/js/loaders/GLTFLoader.js

```diff
@@ -1205,6 +1205,17 @@ THREE.GLTFLoader = ( function () {
 
 		}
 
+		// .extras has user defined data so just in case cheking if .extras.targetNames is array.
```

Comment:
minor: would say `.extras has user-defined data, so check that .extras.targetNames is an array.`

Reply:
Updated. Thanks for always correcting my English!